ocs_ci/ocs/resources/pod.py

```diff
@@ -156,6 +161,25 @@ def get_labels(self):
         """
         return self.pod_data.get('metadata').get('labels')
 
+    def get_pvcname(self):
+        """
+        Get pvcname from pod
+
+        Raises:
+            NotFoundError: If resource not found
+
+        Returns:
+            dict: The openshift pvc_name on a given pod
+        """
+        if self.pod_data.get('spec').get('volumes')[0].get(
+            'persistentVolumeClaim'
+        ) is None:
+            return None
```

Comment:
can't raise a exception here, because this is part of __init__ function which will create issues like below
py", line 173, in get_pvc_name
INTERNALERROR>     ).get('volumes')[0].get('persistentVolumeClaim').get('claimName')
INTERNALERROR> AttributeError: 'NoneType' object has no attribute 'get'

Reply:
removed this part and implemented get_pvc_name as a function, resolving this comment